manim/mobject/mobject.py

```diff
@@ -1309,16 +1309,18 @@ def __init__(self, *mobjects, **kwargs):
 
 
 class _AnimationBuilder:
-    def __init__(self, mobject):
+    def __init__(self, mobject, generate_target=True):
         self.mobject = mobject
+        if generate_target:
+            self.mobject.generate_target()
 
     def __getattr__(self, method_name):
         from ..animation.transform import _MethodAnimation
 
-        self.method = getattr(self.mobject.generate_target(), method_name)
+        self.method = getattr(self.mobject.target, method_name)
```

Comment:
Does this need to be an attribute of `self`? If not then I'd rather it just be `method = ...`